rules/docker-base-gnmi.mk

```diff
@@ -0,0 +1,22 @@
+# docker image for GNMI base
+
+DOCKER_BASE_GNMI_STEM = docker-base-gnmi
+DOCKER_BASE_GNMI = $(DOCKER_BASE_GNMI_STEM).gz
+DOCKER_BASE_GNMI_DBG = $(DOCKER_BASE_GNMI_STEM)-$(DBG_IMAGE_MARK).gz
+
+$(DOCKER_BASE_GNMI)_PATH = $(DOCKERS_PATH)/$(DOCKER_BASE_GNMI_STEM)
+
+$(DOCKER_BASE_GNMI)_DEPENDS += $(SONIC_MGMT_COMMON)
+$(DOCKER_BASE_GNMI)_DEPENDS += $(SONIC_TELEMETRY)
```

Comment:
why base_gnmi depends on telemetry?

Reply:
SONIC_TELEMETRY is the debian package for go binary, I use this rule to install debian package in base image.
https://github.com/sonic-net/sonic-buildimage/blob/master/rules/telemetry.mk#L3

Reply:
Since SONIC_TELEMETRY is just a code name, can we rename SONIC_TELEMETRY -> SONIC_GNMI?